tools/component-generator/templates/_component.scss

```diff
@@ -1,5 +1,12 @@
 // <%= componentName %>
 
```

Comment:
I wonder if we add this into the component generator - so that it's a thing to be deleted, rather than added?

Reply:
I think that's what I've done here? Or did you mean that we should do something else?

Reply:
Gah! My eyes!